main.go

```diff
@@ -106,7 +110,7 @@ func main() {
 			defaultMaxConcurrentReconciles = defaultMaxWorkers
 		}
 	}
-
+	// Why we need a flag for `--leader-election-id`? What is the user case that would be required pass this value?
 	// Deprecated: OPERATOR_NAME environment variable is an artifact of the legacy operator-sdk project scaffolding.
 	//   Flag `--leader-election-id` should be used instead.
 	if operatorName, found := os.LookupEnv("OPERATOR_NAME"); found {
```

Comment:
Why we need a flag for `--leader-election-id`? What is the user case that would be required pass this value?